client/src/components/user/Header/Header.jsx

```diff
@@ -36,7 +36,7 @@ function Header() {
             </button>
             <div className='flex items-center px-2'><input onFocus={() => setSearchBoxFocus(true)} onBlur={() => setSearchBoxFocus(false)} className='outline-none  pl-1 w-80' id='searchBox' placeholder='Search' type="text" /></div>
           </div>
-          <div className='flex border items-center gap-2 border-gray-300 rounded-full py-2 px-4' onClick={(e) => { e.stopPropagation(); setMenu(true) }}>
+          <div className='flex border items-center gap-2 border-gray-300 rounded-full py-2 px-4' onClick={(e) => { e.stopPropagation(); setMenu(!menu) }}>
```

Comment:
Now it is functioning in both directions by reversing the menu state (true/false) to its opposite value.